### PR TITLE
Use Zendesk form for go live requests

### PR DESCRIPTION
**Pull request by klssmith**
The new way of creating support tickets can be seen in [notifications-utils](alphagov/notifications-utils#899).

This changes tickets created when making a request to go live to use the new way, while other tickets stay the same for now.

The go live tags have been removed. Some of these had become unnecessary since you can't make the request to go live unless they are true (e.g. `notify_go_live_email_reply_to`). Others will always get added by a Zendesk macro when the ticket is replied to, so we don't need to add them here.

[Pivotal story](https://www.pivotaltracker.com/story/show/179005041)

(I'm also sneaking a tiny change to remove an unused template, which it doesn't worth putting in its own PR.)

### CODE REVIEW

**Review by idavidmcdonald — COMMENTED** [review]
Looks good.

Requires a change to pip packages required when your other PR is merged.

Also if your other PR does make any changes along the lines of what I optionally suggested then that would equally have knock on effects here that would require changes.

Let me know when you've done any final changes and then I can approve

File: requirements.in [hunk 1]
```diff
@@ -26,7 +26,8 @@ fido2==0.9.1
 awscli-cwlogs>=1.4,<1.5
 itsdangerous==1.1.0  # pyup: <2
 
-git+https://github.com/alphagov/notifications-utils.git@46.0.0#egg=notifications-utils==46.0.0
+# TODO: Update how utils is referenced once the branch is merged
```
[the remark this answers is not in the document]
Comment (klssmith) [reply requirements.in]: TO DO now done

File: requirements.txt [hunk 2]
```diff
@@ -75,7 +75,7 @@ govuk-bank-holidays==0.8
     # via
     #   -r requirements.in
     #   notifications-utils
-git+https://github.com/alphagov/govuk-frontend-jinja.git@v0.5.8-alpha#egg=govuk-frontend-jinja==0.5.8-alpha
+govuk-frontend-jinja @ git+https://github.com/alphagov/govuk-frontend-jinja.git@v0.5.8-alpha
```
Comment (idavidmcdonald) [inline requirements.txt:78]: Some interesting format changing going on here in this file that I haven't seen before. I assume it's OK but feel free to shed any context if you have any.

Maybe something to do with say pip version you may be running locally or something

**Review by idavidmcdonald — APPROVED** [review]
Lovely, nice work :)

File: requirements.txt (hunk 2 above)
Comment (klssmith) [reply requirements.txt:78]: I'm using the latest version of pip locally. I couldn't see exactly which part of the changelog caused the difference in formatting, but there are no breaking changes in the changelog and I think this form should still work with older versions.

**Pull request merged**